### PR TITLE
feat: update project structure to follow changes in Trivet

### PR DESCRIPTION
**Pull request by chosww**
- move output to _site
- update workflows
- add Dependabot configuration
- update markdown config
- add sitemap
- add robots
- update lintings

### CODE REVIEW

File: src/_locales/messages.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// WARNING! This file is generated by a tool.
```
Comment (greatislander) [inline src/_locales/messages.js:1]: I don't think this project has any localized content, so this file and its directory shouldn't be needed.

File: eleventy.config.js [hunk 2]
```diff
@@ -50,31 +48,20 @@ module.exports = function (config) {
...
-                bs.addMiddleware("*", (req, res) => {
-                    const content_404 = fs.readFileSync("dist/404.html");
-                    // Provides the 404 content without redirect.
-                    res.write(content_404);
-                    res.writeHead(404);
-                    res.end();
-                });
-            }
+    config.addPlugin(fluidPlugin, {
+        css: {
```
Comment (greatislander) [inline eleventy.config.js:52]: As this site isn't localized, you should add `i18n: false` to the configuration object here.

File: src/_transforms/parse-transform.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 "use strict";
 const {parseHTML} = require("linkedom");
-const minify = require("../utils/minify.js");
+const minify = require("../_utils/minify.js");
```
Comment (greatislander) [inline src/_transforms/parse-transform.js:3]: Remove this and instead use the same package used in the `minify` transform in `eleventy-plugin-fluid` as it's already a dependency.

**Review by greatislander — CHANGES_REQUESTED** [review]

**Review by greatislander — CHANGES_REQUESTED** [review]
This also needs an RSS feed for news, see #93.

File: src/sitemap.njk [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+permalink: /sitemap.xml
+eleventyExcludeFromCollections: true
+---
+<?xml version="1.0" encoding="utf-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    {% set pages = ["index.html", "404.html", "about.html", "infusion.html", "news/index.html", "projects.html"] %}
```
[the remark this answers is not in the document]
Comment (chosww) [reply src/sitemap.njk:7]: Oh, I didn't know eleventy recognizes htmls outside of collections directory. Thanks for asking, using collections.all works great 👍

**Review by greatislander — CHANGES_REQUESTED** [review]
The 404 file is showing up on the sitemap. To prevent it from showing up there, you need to make sure it is excluded from collections.

File: .github/workflows/release-please.yml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          release-type: node
+          package-name: floeproject.org
```
Comment (greatislander) [inline .github/workflows/release-please.yml:16]: The package name needs to match the project

**Review by greatislander — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 6]
```diff
@@ -1,3 +1,3 @@
 node_modules
-dist
+_site
 package-lock.json
```
Comment (greatislander) [inline .gitignore:3]: The `package-lock.json` file should not be ignore.

**Review by greatislander — APPROVED** [review]

**Pull request merged**